### PR TITLE
Fixes from teaching module 07

### PR DESCRIPTION
**Pull request by jemrobinson**
- Remove duplicated code in linters page
- Remove long quotation
- Use `member variable` throughout
- Explain usage of `_` and `__`

### CODE REVIEW

File: module07_construction_and_design/07_06_classes.ipynb [hunk 1]
```diff
@@ -209,17 +209,16 @@
...
     "Languages provide features to implement this: access control. \n",
     "\n",
     "In python, we use leading underscores to control whether member variables and methods can be accessed from outside the class:\n",
-    " - single leading underscore (`_`) is used to document it's private but people could use it if wanted (thought they shouldn't);\n",
-    " - double leading underscore (`__`) raises errors if called.\n",
     "\n",
-    "\n"
+    "- `_foo`: a single leading underscore (`_`) is a convention to indicate that a variable is private (ie. people are still *able* to use it but they shouldn't)\n",
+    "- `__foo`: a double leading underscore (`__`) is replaced by the interpreter with `_classname__foo` in order to ensure that the name will not overlap with a similar name in another class.\n",
+    "- `__foo__`: this pattern is used by built-in functions like `__init__` or `__str__` so you shouldn't write your own functions with this naming pattern"
```
Comment (jack89roberts) [inline module07_construction_and_design/07_06_classes.ipynb:221]: maybe rephrase slightly, you can write your own `__init__`, `__str__` etc. functions but they should only be for the purposes intended and generally you shouldn't create a `__foo__` method that is not one of the built-ins.

File: module07_construction_and_design/07_06_classes.ipynb [hunk 2]
```diff
@@ -209,17 +209,16 @@
...
     "Robust class design requires consideration of which subroutines are intended for users to use, and which are internal.\n",
     "Languages provide features to implement this: access control. \n",
     "\n",
     "In python, we use leading underscores to control whether member variables and methods can be accessed from outside the class:\n",
-    " - single leading underscore (`_`) is used to document it's private but people could use it if wanted (thought they shouldn't);\n",
-    " - double leading underscore (`__`) raises errors if called.\n",
     "\n",
-    "\n"
+    "- `_foo`: a single leading underscore (`_`) is a convention to indicate that a variable is private (ie. people are still *able* to use it but they shouldn't)\n",
+    "- `__foo`: a double leading underscore (`__`) is replaced by the interpreter with `_classname__foo` in order to ensure that the name will not overlap with a similar name in another class.\n",
```
Comment (radka-j) [inline module07_construction_and_design/07_06_classes.ipynb:220]: it might be good here to be explicit that when you call this variable inside the class, you still use `__foo` but a user trying to access it outside can't, which makes it seem hidden, and they would have to use `_classname__foo` if they are really determined to get to it

Comment (jemrobinson) [reply module07_construction_and_design/07_06_classes.ipynb:220]: Done

File: module07_construction_and_design/07_06_classes.ipynb (hunk 1 above)
Comment (jemrobinson) [reply module07_construction_and_design/07_06_classes.ipynb:221]: Done
